client/src/pages/JoinRoom.tsx

```diff
@@ -1,6 +1,17 @@
 import DisplayLogo from "../components/DisplayLogo";
+import { useState } from "react";
+
+import { socket } from "../App.tsx";
 
 const JoinRoom = () => {
+  // messages
+  const [gameCode, setTournamentCode] = useState("");
+  const [playerName, setPlayerName] = useState("");
+
+  const joinGame = () => {
+    socket.emit("join_game", gameCode, playerName);
```

Comment:
Would prefer to put this string in a constants file.

Reply:
Do you think this is necessary as they are enforced for me? 

![image](https://github.com/Monash-FIT3170/Shazam-3-Scrum-Unleashed/assets/19358855/69d5690e-dd7d-4617-b905-cc122f277d75)